Instructions (#16)
redstagfulfillment/redstagfulfillment.github.io: _posts/2014-07-25-order.md

</tbody>
</table>

<h3 id="order_instruction">
    Order Instruction
</h3>

<table class="table-striped">
<tbody>
<tr>
    <th>note</th>
    <td>
        <pre><code>{ "note" : "Place Amazon FBA Label in a pouch" }</code></pre>
        The "Note" property.
    </td>
</tr>
<tr>
    <th>file_name</th>
    <td>
        <pre><code>{ "file_name" : "amazon_fba_3425232.pdf" }</code></pre>
        The "File Name" property.
    </td>
</tr>
<tr>
    <th>file_content</th>
    <td>
        <pre><code>{ "file_content" : "base64 encoded file content" }</code></pre>
        The base64 encoded "File Content" property.
    </td>
</tr>
<tr>

>     <th>print_copies</th>

Reviewer: ```
    <th>presentation</th>
```

>     <th>print_copies</th>

Reviewer: ```suggestion
    <th>presentation</th>
```

>     <th>print_copies</th>

Reviewer: ```suggestion
    <th>presentation</th>
```

    <td>

>         <pre><code>{ "print_copies" : "one_per_order" }</code></pre>

Reviewer: <pre><code>{ "presentation" : "once_per_order" }</code></pre>

>         <pre><code>{ "print_copies" : "one_per_order" }</code></pre>

Reviewer: ```suggestion
        <pre><code>{ "presentation" : "once_per_order" }</code></pre>
```

>         The "Print Copies" property. Values can be <code>one_per_order</code>, <code>one_per_shipment</code>, <code>one_per_package</code>, or <code>none</code>.

Reviewer: ```suggestion
        The "Presentation" property. Values can be <code>once_per_order</code>, <code>once_per_shipment</code>, or <code>once_per_package</code>.
```

    </td>
</tr>
<tr>
    <th>print_target</th>
    <td>

>         <pre><code>{ "print_target" : "4X6_LABEL" }</code></pre>

Reviewer: ```suggestion
        <pre><code>{ "print_target" : "LABEL" }</code></pre>
```

>         The "Print Target" property. Values can be <code>4X6_LABEL</code>, <code>SMALL_LABEL</code>, or <code>LASER</code>.

Reviewer: ```suggestion
        The "Print Target" property. Values can be <code>LABEL</code>, <code>SMALL_LABEL</code>, or <code>LASER</code>.
```
